firmware/examples/Post/source/main.cpp

```diff
@@ -0,0 +1,287 @@
+#include <cstdarg>
+#include <cstdint>
+#include <cstdio>
+#include <cstdlib>
+
+#include "L0_LowLevel/LPC40xx.h"
+#include "L1_Drivers/gpio.hpp"
+#include "L1_Drivers/i2c.hpp"
+#include "L1_Drivers/ssp.hpp"
+#include "L2_Utilities/log.hpp"
+#include "L2_Utilities/status.hpp"
+#include "L3_HAL/onboard_led.hpp"
+#include "L3_HAL/ssd1306.hpp"
+#include "config.hpp"
+
+constexpr uint8_t kFirstI2cAddress = 0x08;
+constexpr uint8_t kLastI2cAddress  = 0x78;
+
+constexpr uint8_t kAccelerometerAddress = 0x1C;
+constexpr uint8_t kGestureAddress       = 0x39;
+constexpr uint8_t kTemperatureAddress   = 0x31;
+
+I2c i2c;
+
+bool ExternalFlashTest();
+bool I2cTest();
+void OnBoardLedTest(bool gpio_test, bool i2c_test, bool flash_test);
+bool GpioFunctionalityTest();
+bool GestureSensorTest();
+bool AccelerometerTest();
+bool TemperatureSensorTest();
+void OledTest();
+
+int main(void)
+{
+  OledTest();
+  bool gpio_pass  = GpioFunctionalityTest();
+  bool i2c_pass   = I2cTest();
+  bool flash_test = ExternalFlashTest();
+  OnBoardLedTest(gpio_pass, i2c_pass, flash_test);
+
+  // LOG_INFO("Running loop.");
+  Delay(5000);
+  return 0;
+}
+
+void OledTest()
+{
+  Ssd1306 display;
+  LOG_INFO("Starting OLED Hardware Test...");
+
+  LOG_INFO("Initializing OLED Hardware Test...");
+  display.Initialize();
+
+  LOG_INFO("Clearing Screen...");
+  display.Clear();
+
+  LOG_INFO("Clearing Screen finished. Waiting 1 second...");
+
+  Delay(1000);
```

Comment:
This is actually not required. Should work if you remove this.

You can remove the "Waiting 1 second..." part of hte message above.